docs/access-management/sso-providers/azure-entra-id.md

```diff
@@ -87,6 +87,10 @@ To create client credentials for Airbyte to interface with your application, hea
 1. Click **New client secret**, and enter the expiry date of your choosing. You'll need to pass in the new client secret every time the old one expires to continue being able to log in via Entra ID.
 2. Copy the **Value** (the client secret itself) immediately after creation. You won't be able to view this later on.
 
+:::caution
+If you want to avoid admin consent for these permissions, be sure to choose "Delegated permissions" that do not require admin consent for common scopes. Application permissions will specifically require admin approval.
```

Comment:
Can you add a screenshot here? Since this is super important

Reply:
Yeah will do